### PR TITLE
stat_prop(), ggally_colbar() & ggbivariate()

### PR DESCRIPTION
**Pull request by larmarange**
A set with:

- `stat_prop()` as discussed here #317 
- `ggally_colbar()` & `ggally_rowbar()` for barplots displaying column and row proportions
- `ggbivariate()`, a variation of `ggdduo()` dedicated to plots variations of and outcome according to explanory variables (see the examples)

### CODE REVIEW

**Comment by larmarange** [comment]
The code of `ggbivariate()` could be updated once #331 accepted

File: R/stat_prop.R [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+   panel
+  }
+)
+
+
+"%||%" <- function(a, b) {
+  if (!is.null(a)) a else b
+}
+
```
Comment (schloerke) [inline R/stat_prop.R:130]: This is already imported from rlang
```suggestion
"%||%" <- function(a, b) {
  if (!is.null(a)) a else b
}

```

Comment (larmarange) [reply R/stat_prop.R:130]: ok fixed

**Comment by larmarange** [comment]
I have updated the PR. It should be ok for merging now.

**Pull request merged**